src/Microsoft.SqlTools.ServiceLayer/Localization/sr.cs

```diff
@@ -205,6 +205,14 @@ public static string QueryServiceColumnNull
             }
         }
 
+        public static string QueryServiceCellNull
+        {
+            get
+            {
+                return Keys.GetString(Keys.QueryServiceCellNull);

```

Comment:
Pretty much so that we always return the localized null string and so that we always return *something* for the display value of a dbcellvalue.

Though the code that actually changes behavior is in the "Changes that *actually do something!*" v2 commit... whoops